Dispatcher.php

```diff
@@ -91,6 +107,12 @@ public function send(Message $message, $templateName = '', $templateContent = ar
             return false;
         }
 
+        if ($this->useProxy()) {
+            $this->addCurlProxyOptions();
+        }
+
+        $this->addCurlSSLOptions();
+
```

Comment:
It would be great if you move the call of addCurlProxyOption and addCurlSSLOptions to constructor.